README.md

```diff
@@ -50,6 +50,15 @@ Pull requests with additional tools and projects are more than welcome!
 - [Roblox/nomad-node-problem-detector](https://github.com/Roblox/nomad-node-problem-detector) - A tool used to detect problems on Nomad nodes based on user-defined health checks.
 - [hashicorp/nomad-pack](https://github.com/hashicorp/nomad-pack) - An official templating tool and package manager for Nomad, currently a Tech Preview.
 - [dmaes/nomad-logger](https://github.com/dmaes/nomad-logger) - Watch Nomad allocations and update logshipper config.
+- [kamilcuk/nomad-tools](https://github.com/Kamilcuk/nomad-tools) - Collection of the following tools:
```

Comment:
please keep it at one line with the tagline to keep it fair with other projects :)

Reply:
Sure, I will reformat.